flink-dist/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,62 @@
+flink-dist
+Copyright 2014-2018 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies under the Apache Software License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+- com.twitter:chill-java:0.7.6
+- com.twitter:chill_2.11:0.7.6
+- com.typesafe:config:1.3.0
+- com.typesafe:ssl-config-core_2.11:0.2.1
+- com.typesafe.akka:akka-actor_2.11:2.4.20
+- com.typesafe.akka:akka-camel_2.11:2.4.20
+- com.typesafe.akka:akka-protobuf_2.11:2.4.20
+- com.typesafe.akka:akka-slf4j_2.11:2.4.20
+- com.typesafe.akka:akka-stream_2.11:2.4.20
+- commons-cli:commons-cli:1.3.1
+- commons-collections:commons-collections:3.2.2
+- commons-io:commons-io:2.4
+- org.apache.camel:camel-core:2.17.7
+- org.apache.commons:commons-compress:1.4.1
+- org.apache.commons:commons-lang3:3.3.2
+- org.apache.commons:commons-math3:3.5
+- org.javassist:javassist:3.19.0-GA
+- org.objenesis:objenesis:2.1
+- org.rocksdb:rocksdbjni:5.7.5
+- org.xerial.snappy:snappy-java:1.1.4
+
+This project bundles the following dependencies under the BSD license.
+See bundled license files for details.
+
+- org.clapper:grizzled-slf4j_2.11:1.3.2
+
+This project bundles the following dependencies under the BSD 3-Clause license.
+See bundled license files for details.
+
+- com.esotericsoftware.kryo:kryo:2.24.0
+- com.esotericsoftware.minlog:minlog:1.2
+
+The following dependencies all share the same BSD 3-Clause license which you find under licenses/LICENSE.scala.
+
+- org.scala-lang:scala-compiler:2.11.12
+- org.scala-lang:scala-library:2.11.12
+- org.scala-lang:scala-reflect:2.11.12
+- org.scala-lang.modules:scala-java8-compat_2.11:0.7.0
+- org.scala-lang.modules:scala-parser-combinators_2.11:1.0.4
+- org.scala-lang.modules:scala-xml_2.11:1.0.5
+
+This project bundles the following dependencies under the MIT/X11 license.
+See bundled license files for details.
+
+- com.github.scopt:scopt_2.11:3.5.0
+- org.slf4j:slf4j-api:1.7.15
+
+This project bundles the following dependencies under the Public Domain.
```

Comment:
I would add the full notice text from the project:
```
This project bundles "org.tukaani xz version 1.5".
This Java implementation of XZ has been put into the public domain, thus you can do
whatever you want with it. All the files in the package have been written by Lasse Collin,
but some files are heavily based on public domain code written by Igor Pavlov.
```

Reply:
That text is the license text from the project homepage.

Reply:
Yes, this is definitely missing. Will add it.